pkg/goversion/compat.go

```diff
@@ -5,10 +5,12 @@ import (
 )
 
 var (
-	minSupportedVersionOfGoMinor = 11
-	maxSupportedVersionOfGoMinor = 13
-	goTooOldErr                  = fmt.Errorf("Version of Go is too old for this version of Delve (minimum supported version 1.%d, suppress this error with --check-go-version=false)", minSupportedVersionOfGoMinor)
-	dlvTooOldErr                 = fmt.Errorf("Version of Delve is too old for this version of Go (maximum supported version 1.%d, suppress this error with --check-go-version=false)", maxSupportedVersionOfGoMinor)
+	MinSupportedVersionOfGoMajor = 1
+	MinSupportedVersionOfGoMinor = 11
+	MaxSupportedVersionOfGoMajor = 1
+	MaxSupportedVersionOfGoMinor = 13
+	goTooOldErr                  = fmt.Errorf("Version of Go is too old for this version of Delve (minimum supported version 1.%d, suppress this error with --check-go-version=false)", MinSupportedVersionOfGoMinor)
```

Comment:
nit: please use new major version const in string interpolation.

Reply:
Done.